SH4RE/SH4RE/ContentView.swift

```diff
@@ -58,7 +59,7 @@ struct ContentView: View {
                             Label("Messages", systemImage: "message.fill")
                         }
                         .tag(4)
-                    AccountView(tabSelection: $tabSelection)
+                    AccountView(tabSelection: $tabSelection, favouritesModel: favouritesModel)
```

Comment:
can you get rid of this, I will add it in an account page favourties PR

Reply:
I guess... the conflict will be so minimal that I don't really think this is worth it, but ok.

Reply:
favourites will not be able to be implemented by bryan until accounts page is done if I am removing this.